core/src/main/java/com/squarespace/template/plugins/platform/CommerceFormatters.java

```diff
@@ -466,18 +468,67 @@ private static String defaultSubscriptionPriceString(JsonNode productNode) {
 
       return sb.toString();
     }
+  }
 
-    private static String getMoneyString(JsonNode moneyNode, Context ctx) {
```

Comment:
Because this is used in both `product-price` and `subscription-price` I moved it out to `CommerceUtils`